src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractClassNameCheck.java

```diff
@@ -33,9 +33,9 @@
  * </p>
  * <p>
  * Rationale: Abstract classes are convenience base class implementations of
- * interfaces, not types as such. As such they should be named to indicate this.
- * Also, if names of classes starts with 'Abstract' it's very convenient that
- * they will have abstract modifier.
+ * interfaces, and are not considered types per se. For this reason, it should
```

Comment:
> considered types per se

I do not understand this this English.  please rephrase in simple way, that will be clearly understood by any non-English people.

you might just need to remove ", not types as such."

Reply:
How about the below one? 

Rationale: Abstract classes are convenience base class implementations of
interfaces. For this reason, it should be made obvious that a given class is 
abstract by prefacing the class name with 'Abstract'.

Reply:
Good

Reply:
done